packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -0,0 +1,519 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { Resource, Construct, Duration, Token, Annotations, RemovalPolicy, IResource } from '@aws-cdk/core';
+import { IClusterEngine } from './cluster-engine';
+import { DatabaseSecret } from './database-secret';
+import { Endpoint } from './endpoint';
+import { IParameterGroup } from './parameter-group';
+import { applyRemovalPolicy, DEFAULT_PASSWORD_EXCLUDE_CHARS } from './private/util';
+import { Credentials, RotationMultiUserOptions, RotationSingleUserOptions } from './props';
+import { CfnDBCluster } from './rds.generated';
+import { ISubnetGroup, SubnetGroup } from './subnet-group';
+
+/**
+ *  Properties to configure an Aurora Serverless Cluster
+ */
+export interface ServerlessClusterProps {
```

Comment:
A lot of those are shared with `DatabaseClusterProps`. Can you extract a module-private interface that they both inherit from, to reduce duplication?

Reply:
gave this a shot, but `jsii` complains with:
> Unable to resolve type "@aws-cdk/aws-rds.CommonDatabaseClusterProps". It may be @iternal or not exported from the module's entry point (as configured in "package.json" as "main").

talked about this offline and we decided we would leave this alone for now.